src/data/resolvers/motions.ts

```diff
@@ -0,0 +1,110 @@
+import { Resolvers } from '@apollo/client';
+
+import {
+  ClientType,
+  getLogs,
+  getBlockTime,
+  ExtensionClient,
+} from '@colony/colony-js';
+
+import { Context } from '~context/index';
+import { ColonyAndExtensionsEvents } from '~types/index';
+import {
+  ActionsPageFeedType,
+  SystemMessage,
+  SystemMessagesName,
+} from '~dashboard/ActionsPageFeed';
+import { NetworkMotionState } from '~utils/colonyMotions';
```

Comment:
This should be imported from `colonyJS` . It's available since version `4.0.0-rc.0`

```suggestion
import { MotionState as NetworkMotionState } from '@colony/colony-js';
```